pkgs/tools/system/lanzaboote/stub.nix

```diff
@@ -0,0 +1,31 @@
+{ rustPlatform, uefiLinker ? null, stdenv, lib, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage
+rec {
+  pname = "lanzaboote_stub";
+  version = "0.3.0";
+  src = fetchFromGitHub {
+    owner = "nix-community";
+    repo = "lanzaboote";
+    rev = "v${version}";
+    hash = "sha256-Fb5TeRTdvUlo/5Yi2d+FC8a6KoRLk2h1VE0/peMhWPs=";
+  };
+
+  sourceRoot = "source/rust/stub";
+  cargoHash = "sha256-uj7+X0mQxKfbLcSbg4SfHFyp2SHIymwBDzDFvJdcGD8=";
+
+  # Necessary because our `cc-wrapper` doesn't understand MSVC link options.
+  # RUSTFLAGS = "-Clinker=${stdenv.cc.bintools}/bin/${stdenv.cc.targetPrefix}lld-link";
+  RUSTFLAGS = if uefiLinker != null then "-Clinker=${uefiLinker}/bin/lld-link" else "";
```

Comment:
OK, here, it's kinda ugly, @amjoseph-nixpkgs 
First, I'd like to avoid passing `RUSTFLAGS = "-Clinker=${uefiLinker}/bin/lld-link"` by tucking this inside a UEFI stdenv, does that sound OK for you?